packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -164,8 +164,12 @@ export const IndexesToolbar: React.FunctionComponent<IndexesToolbarProps> = ({
                 className={alignSelfEndStyles}
                 label="Viewing"
                 defaultValue="regular-indexes"
+                data-testid="indexes-segment-controls"
               >
-                <SegmentedControlOption value="regular-indexes">
+                <SegmentedControlOption
+                  data-testid="regular-indexes"
```

Comment:
These test ideas are bound to clash eventually when we lookup a test id at a page level. ie. quite possibly with the list or something. maybe view-regular-indexes-option and view-search-indexes-option or something like that?